input/prometheus/prometheus.go

```diff
@@ -0,0 +1,138 @@
+package prometheus
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/golang/snappy"
+	"github.com/grafana/grafana/pkg/log"
```

Comment:
use `github.com/raintank/worldping-api/pkg/log` instead for now. it's the library we use everywhere else in MT, and is already vendored in. unless you have a compelling reason i missed?